src/components/SelectButtonGroup/SelectButton/SelectButton.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+
+interface Props {
+    /** the label to be displayed on this button */
+    children: string;
+    /** the value associated with this button */
+    value: string;
+    /**
+     * A function to be called if a button is pressed.
+     * This prop, if defined on the parent SelectButtonGroup will overwrite this one
+     */
+    onSelect?: (value: string) => void;
+    /** if this button should be in a selected state */
+    isSelected?: boolean;
```

Comment:
Is this a suitable interface in case of a single select (radios)? Can you make sure there is only one value selected or is that up to the integrating application? For single selects what you typically see is e.g. `value` (for specifying selected value) on the outer component (`SelectButtonGroup` in this case) but on the other hand, that's probably less suitable for multi select.

Reply:
in native HTML you have `checked` both on radio button and checkbox. So I think it will work here too